moto/awslambda/models.py

```diff
@@ -735,7 +741,12 @@ def update_from_cloudformation_json(
     def _create_zipfile_from_plaintext_code(code):
         zip_output = io.BytesIO()
         zip_file = zipfile.ZipFile(zip_output, "w", zipfile.ZIP_DEFLATED)
-        zip_file.writestr("lambda_function.zip", code)
+        zip_file.writestr("index.py", code)
+        # This should really be part of the 'lambci' docker image
+        from moto.packages.cfnresponse import cfnresponse
```

Comment:
Same scenario: this comes into play when creating a Lambda function via CF with the code provided inline in the CF template.

From https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html:

>
> The cfn-response module is available only when you use the ZipFile property to write your source code. It isn't available for source code that's stored in Amazon S3 buckets. For code in buckets, you must write your own functions to send responses. 
>